plugins/woocommerce/src/Admin/API/OnboardingPlugins.php

```diff
@@ -95,6 +96,32 @@ function ( $value ) {
 				'schema' => array( $this, 'get_install_async_schema' ),
 			)
 		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/jetpack-authorization-url',
```

Comment:
Could we add a description about this endpoint being experimental (so we can make breaking changes later)? We haven't had much time to think about this API in totality.

According to ChatGPT we can add a description to `args` but I'm not sure about that 🤔 

```
'args' => array(
            'description' => 'This is a custom endpoint that does XYZ.',
        ),
```

Reply:
Godo suggestion. Updated in https://github.com/woocommerce/woocommerce/pull/38674/commits/0e7d7bd9f825264614dfa792a988b9ed92414f34

Added a comment instead. I think ChatGPT still needs to learn more about WordPress syntax 😄